### PR TITLE
feat: add introduction videos on signup

### LINKED ISSUE #1580: Add intro videos in the sign up process
Currently when signing up a new user, there is a page which is empty and says 'intro video'.

### PR DESCRIPTION
**Pull request by chosww**
Resolves #1580  

### Prerequisites

If this PR changes PHP code or dependencies:

- [x] I've run `composer format` and fixed any code formatting issues.
- [x] I've run `composer analyze` and addressed any static analysis issues.
- [x] I've run `php artisan test` and ensured that all tests pass.
- [x] I've run `composer localize` to update localization source files and committed any changes.

If this PR changes CSS or JavaScript code or dependencies:

- [x] I've run `npm run lint` and fixed any linting issues.
- [x] I've run `npm run build` and ensured that CSS and JavaScript assets can be compiled.

### CODE REVIEW

File: resources/views/users/show-introduction.blade.php [hunk 1]
```diff
@@ -12,9 +12,25 @@
...
+                speed: true,
+                title: false
+            })" @ended="player().setCurrentTime(0)">
+            </div>
+        </div>
+    @else
+        <div class="frame">
+            {{ __('Introduction video') }}
+        </div>
+    @endif
```
Comment (jobara) [inline resources/views/users/show-introduction.blade.php]: I wonder if we should fall back to a default language if a locale doesn't exist. Maybe the English one? Which is what I think happens if the site isn't localized in a given language. I suppose if the `locale()` and `en` don't exist, then we don't include anything, including the `h2` that says "Introduction video` and just leave the site to say welcome and the Continue button at the bottom. 

Does that seam reasonable? Let me know what you think, or if I'm missing anything.

Comment (chosww) [reply resources/views/users/show-introduction.blade.php]: Thanks @jobara, your suggestion sounds good to me and would simplify code blocks. I will update it like your suggestion.

File: app/Models/User.php [hunk 2]
```diff
@@ -185,14 +185,28 @@ public function participantInvitations(): Collection
...
-            'training-participant' => __('Video for training participants.'),
-            default => '',
+            'individual' => [
+                'en' => 'https://vimeo.com/850314990/05587fe4df?share=copy',
+                'fr' => 'https://vimeo.com/850319076/4d973fc4ee?share=copy',
+                'asl' => 'https://vimeo.com/850314990/05587fe4df?share=copy',
+                'lsq' => 'https://vimeo.com/850322469/cd5616567a?share=copy',
+            ],
+            'organization' => [
+                'en' => 'https://vimeo.com/850315035/87b6129a8b?share=copy',
```
Comment (jobara) [inline app/Models/User.php]: The URL for the video is incorrect as it's a duplicate of the ASL video. The correct URL is `https://vimeo.com/manage/videos/850308900/39c5bb60a7`

File: app/Models/User.php [hunk 3]
```diff
@@ -185,14 +185,28 @@ public function participantInvitations(): Collection
...
+    public function introduction(): array
     {
         return match ($this->context) {
-            'individual' => __('Video for individuals.'),
-            'organization' => __('Video for community organizations.'),
-            'regulated-organization' => __('Video for regulated organizations.'),
-            'training-participant' => __('Video for training participants.'),
-            default => '',
+            'individual' => [
+                'en' => 'https://vimeo.com/850314990/05587fe4df?share=copy',
```
Comment (jobara) [inline app/Models/User.php]: The URL for the video is incorrect as it's a duplicate of the ASL video. The correct URL is `https://vimeo.com/850308866/22cf4718fc?share=copy`

File: app/Models/User.php [hunk 4]
```diff
@@ -185,14 +185,28 @@ public function participantInvitations(): Collection
...
+                'lsq' => 'https://vimeo.com/850322469/cd5616567a?share=copy',
+            ],
+            'organization' => [
+                'en' => 'https://vimeo.com/850315035/87b6129a8b?share=copy',
+                'fr' => 'https://vimeo.com/850319102/c118d69046?share=copy',
+                'asl' => 'https://vimeo.com/850315035/87b6129a8b?share=copy',
+                'lsq' => 'https://vimeo.com/850322511/2aad27699a?share=copy',
+            ],
+            'regulated-organization' => [
+                'en' => 'https://vimeo.com/850315068/bc26c699cb?share=copy',
```
Comment (jobara) [inline app/Models/User.php]: The URL for the video is incorrect as it's a duplicate of the ASL video. The correct URL is `https://vimeo.com/850308924/cab1e34418?share=copy`

**Review by jobara — CHANGES_REQUESTED** [review]

**Review by jobara — APPROVED** [review]

**Pull request merged**